packages/java/engine-core/src/main/java/com/vaadin/hilla/engine/EngineConfiguration.java

```diff
@@ -20,6 +20,7 @@
 public class EngineConfiguration {
     public static final String DEFAULT_CONFIG_FILE_NAME = "hilla-engine-configuration.json";
     public static final String OPEN_API_PATH = "hilla-openapi.json";
+    public static final String FILE_ROUTES_PATH = "file-routes.json";
```

Comment:
I believe we don't need the constant for the `file-routes.json` path to be defined in this class. Use `ClientRouteRegistry.FILE_ROUTES_JSON_PROD_PATH` instead.